_hackbars/vishnur.md

```diff
@@ -0,0 +1,17 @@
+---
+layout : hackbar
+title  : hacktoberfest
+autor  : vishnur
+---
+
+
+open source is the best source to show one's talent.
+
+
+---
+
+## Leave some greenery for the future to enjoy the Scenery..
+
+---
+
+![Melting_Earth]({{site.baseurl}}/assets/images/vishnur.jpg)
```

Comment:
hey @Vishnurr2k01 Welcome to Hacktoberfest-2020
Can you please add the missing image `vishnur.jpg`?